packages/storage/package.json

```diff
@@ -10,9 +10,11 @@
     "dist"
   ],
   "scripts": {
+    "lint": "eslint -c .eslintrc.json '**/*.ts' --ignore-path '../../.gitignore'",
+    "lint:fix": "eslint --fix -c .eslintrc.json '**/*.ts'",
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "test": "run-p test:browser",
+    "test": "run-p lint test:browser",
```

Comment:
Note: With this change, it's no longer possible to add `it.only` and run the tests, since the linter will reject the change before the tests run.

Reply:
We can add a new command `"test:no-lint": "run-p test:all"` to facilitate local development.

Reply:
Can we just flip them? It allows to run the tests, but still prevents us from checking this in.

Reply:
Do you mean make `yarn test` not lint, but make `yarn test:lint` lint?
It's supposed to prevent us from checking code in if any lint rule fails, so CI should always run the version that does linting.
Our CI run `yarn test`, so I'm making lint part of it. Am I understanding your concern correctly?

Reply:
This is what I mean:

```
"test": "run-p test:browser lint"
```

That way we run the tests before we fail the lint step. I can ignore the failed lint step in my local development.

Reply:
Ah, yes. I thought about it, but decided to lint first because I want it to fail fast - tests take much longer to finish.
I'm prioritizing CI over local development because travis is very slow and is time consuming to restart. Imagine running tests for 10 minutes only to find that lint failed.

That being said - I think it's okay to flip the order of lint and test since you will run lint locally anyway (just later), so lint should always pass in CI. Though I worry sometimes people forget to run test at all before committing changes, it should be rare.

so.. I updated the test command to `"test": "run-p test:browser lint"`